diff --git a/include/riglogic/version/Version.h b/include/riglogic/version/Version.h
index 81a1d5e..d752b76 100644
--- a/include/riglogic/version/Version.h
+++ b/include/riglogic/version/Version.h
@@ -4,5 +4,5 @@
 
 #define RL_MAJOR_VERSION 13
 #define RL_MINOR_VERSION 2
-#define RL_PATCH_VERSION 8
-#define RL_VERSION_STRING "13.2.8"
+#define RL_PATCH_VERSION 7
+#define RL_VERSION_STRING "13.2.7"
